src/impact_analysis.py

```diff
@@ -61,12 +61,16 @@ def determine_changed_dbt_models() -> List[DbtNodeInfo]:
                 "ls",
                 # fmt: off
```

Comment:
hm the fmt:off and on were supposed to keep the formatted as-is here. not sure why that didn't work. it's not critical - but would be good to remove the `fmt: off` fences if they're not being used